Cargo.toml

```diff
@@ -17,10 +17,10 @@ description = "A standalone implementation of the RedJubjub signature scheme."
 features = ["nightly"]
 
 [dependencies]
-reddsa = "0.5.0"
-rand_core = "0.6"
+reddsa = { version = "0.5.0", default-features = false, features = ["alloc"] }
+rand_core = { version = "0.6", default-features = false }
 serde = { version = "1", optional = true, features = ["derive"] }
```

Comment:
This should also have `default-features = false` to ensure that enabling the `serde` feature flag doesn't force `std` to also be enabled.